internal/storage/bucket_handle.go

```diff
@@ -0,0 +1,58 @@
+package storage
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/storage"
+	"github.com/jacobsa/gcloud/gcs"
+	"google.golang.org/api/iterator"
+)
+
+type bucketHandle struct {
+	gcs.Bucket
+	bucket *storage.BucketHandle
+}
+
+//func(bh *bucketHandle) Name string {
+//}
+
+func (bh *bucketHandle) NewReader(
+		ctx context.Context,
+		req *gcs.ReadObjectRequest) (rc io.ReadCloser, err error) {
+	// make a call to gcs
+	return
+}
+func (b *bucket) DeleteObject(
+		ctx context.Context,
+		req *DeleteObjectRequest) (err error) {
```

Comment:
no need to add every parameter on a new line. keep the function definition on one line.
https://github.com/golang/go/wiki/CodeReviewComments#line-length

Reply:
fixed